testing/test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
+while getopts ":h" option; do
+   case $option in
+      h) # display Help
+         echo "This is a script for running the R-judge local"
+         echo
+         echo "Syntax: test.sh [-h] PATH_TO_EXERCISE_DIR PATH_TO_DUMP_DIR [PATH_TO_SOLUTION_FILE]"
+         echo "Param: \n\
```

Comment:
`\n\` basically puts a newline, and then escapes the automatic newline, so you can just drop this.